lib/media/helper_modules/s3_manager.ex

```diff
@@ -227,11 +227,11 @@ defmodule Media.S3Manager do
     end
   end
 
-  """
-  This function toggles the object privacy.
-  It takes the object key as a first argument and the new privacy status as a second argument
-  The object key is the object filename.
-  """
+  # """
```

Comment:
why are you commenting out this doc. ?

Reply:
Internal functions no need to expose their usage as they should not be used within the host project. So I just put the comments in order to clarify the functionality.